### PR TITLE
feature: create the widget in ts for trustpilot with default value fo…

### PR DESCRIPTION
**Pull request by loicmasson2**
…r happeo and free tier template. Create the readme with relevant informations

It should work both when you run outside a page and independently.

### CODE REVIEW

File: examples/TrustPilotWidget/package.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    "@babel/preset-env": "^7.14.5",
+    "@babel/preset-react": "^7.14.5",
+    "@happeouikit/alerts": "latest",
+    "@happeouikit/avatar": "latest",
+    "@happeouikit/banners": "latest",
+    "@happeouikit/buttons": "latest",
+    "@happeouikit/card": "latest",
+    "@happeouikit/colors": "latest",
+    "@happeouikit/content-renderer": "latest",
+    "@happeouikit/copy-to-clipboard-button": "latest",
```
Comment (zaepon) [inline examples/TrustPilotWidget/package.json]: @happeouikit/content-renderer": "latest",
@happeouikit/copy-to-clipboard-button": "latest",

These happeouikit packages are private deps and require auth.
We recently removed these from other examples

File: examples/TrustPilotWidget/package.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    "webpack-dev-server": "^3.11.2"
+  },
+  "peerDependencies": {
+    "@happeouikit/alerts": "latest",
+    "@happeouikit/avatar": "latest",
+    "@happeouikit/banners": "latest",
+    "@happeouikit/buttons": "latest",
+    "@happeouikit/card": "latest",
+    "@happeouikit/colors": "latest",
+    "@happeouikit/content-renderer": "latest",
```
Comment (zaepon) [inline examples/TrustPilotWidget/package.json]: ^ check above comment

File: examples/TrustPilotWidget/src/TrustPilot.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    // Create a reference to the <div> element which will represent the TrustBox
+    const ref = React.useRef(null);
+    React.useEffect(() => {
+        // If window.Trustpilot is available it means that we need to load the TrustBox from our ref.
+        // If it's not, it means the script you pasted into <head /> isn't loaded  just yet.
+        // When it is, it will automatically load the TrustBox.
+        if (window.Trustpilot) {
+            window.Trustpilot.loadFromElement(ref.current, true);
+        }
+    }, []);
```
Comment (zaepon) [inline examples/TrustPilotWidget/src/TrustPilot.tsx]: should businessUnitId and templateId be added to the useEffect dependencies ? 

`}, [businessUnitId, templateId]);`

This way the trustpilot component will re-render when businessUnitId or templateId are changed in the settings and the change will be visible instantly without needing to save&reload the page.

File: examples/TrustPilotWidget/package.json (hunk 2 above)
Comment (zaepon) [reply examples/TrustPilotWidget/package.json]: here too :)

File: examples/TrustPilotWidget/package.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
+{
+  "name": "trustpilotwidget",
```
Comment (mikamaunula) [inline examples/TrustPilotWidget/package.json]: Could it be named `trustpilot-widget` to follow npm good naming practises

File: examples/TrustPilotWidget/src/Widget.tsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        return () => {
+            document.body.removeChild(script);
+        }
+    }, [id]);
+
+    if (!initialized) {
+
+        return (
+            <>
+                <h3>Loading ...</h3>
```
Comment (mikamaunula) [inline examples/TrustPilotWidget/src/Widget.tsx]: Please use Loader component from kit.

**Review by mikamaunula — APPROVED** [review]

**Review by zaepon — APPROVED** [review]

**Pull request merged**